apps/test-blinky/src/main.cpp

```diff
@@ -0,0 +1,22 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2019 ARM Limited
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "mbed.h"
+
+static BufferedSerial pc(SERIAL_TX, SERIAL_RX, ROVER_DEFAULT_SERIAL_BAUD_RATE);
```

Comment:
This changes the printf to be going to the serial tx and rx instead of the default (usb serial tx and rx)

Reply:
iirc they're the same on nucleo and different elsewhere but i may be wrong

Reply:
Was just confused because u didnt use the `pc` object?

Default is always STDIO_UART_TX/RX unless MBED_CONF_TARGET_STDIO_UART_TX/RX is defined. This is intended for `printf` debug output so `pc` is only needed if intent is to use for purpose other than debug.

Reply:
I don't really like the definition of the USBTX/RX pins anyways, they are redundant and idk why they are there... probably good to just remove them.

Reply:
Does ` STDIO_UART_TX/RX` equal `SERIAL_TX, SERIAL_RX`? 

the `pc.printf` got deprecated. afaik, printf uses the default like u said, unless theres a Buffered serial or unbufferedserial in scope... then it automatically uses the scoped serial thing

Reply:
> I don't really like the definition of the USBTX/RX pins anyways, they are redundant and idk why they are there... probably good to just remove them.

do they alias to smth else? afaik they only make sense for nucleo

Reply:
imo `printf` should be used for all standard console prints so they all use the same setting, and all standard console prints should always use STDIO_UART_TX/RX defined for the target.

Reply:
Do the apps work rn without the buffered serial object? maybe i was mistaken about which pins were the default and the default output is already routed to the right place

Reply:
Yes